src/index.js

```diff
@@ -20,6 +20,10 @@ ReactDOM.render(
   </BrowserRouter>,
   document.getElementById('root'),
 );
-registerServiceWorker();
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.register();
```

Comment:
I think the default now is to set it to `unregister()`. I think that's probably better for this demo and then if someone wants to register the service worker they can change it locally.